internal/rukpak/source/containers_image.go

```diff
@@ -89,7 +85,11 @@ func (i *ContainersImageRegistry) Unpack(ctx context.Context, bundle *BundleSour
 	if err != nil {
 		return nil, fmt.Errorf("error creating temporary directory: %w", err)
 	}
-	defer os.RemoveAll(layoutDir)
+	defer func() {
+		if err := os.RemoveAll(layoutDir); err != nil {
```

Comment:
Should this be `deleteRecursive()`?

Reply:
This one doesn't have to be because `layoutDir` only ever exists while this function is running, and we never set it to be read-only like we do for the unpack dir.